package.json

```diff
@@ -99,7 +100,9 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.2.5",
     "phantomjs": "^1.9.17",
+    "random-words": "0.0.1",
     "react": "^0.14.0",
+    "react-addons-perf": "^0.14.7",
```

Comment:
I think that we can remove this dependency from the package.json of the `docs` now.